web/modules/custom/joinup_front_page/src/Plugin/Block/ExploreItemBlock.php

```diff
@@ -0,0 +1,203 @@
+   */
+  public function build(): array {
+    $data = [
+      'solutions' => [
+        'name' => $this->t('Solutions'),
+        'data' => $this->getRdfEntity('solution'),
+        'url' => '/search?keys=&f[0]=type%3Asolution',
+      ],
+      'collections' => [
+        'name' => $this->t('Collections'),
+        'data' => $this->getRdfEntity('collection'),
+        'url' => '/search?keys=&f[0]=type%3Acollection',
+      ],
+      'news' => [
+        'name' => $this->t('News'),
+        'data' => $this->getContent('news'),
+        'url' => '/search?keys=&f[0]=type%3Anews',
+      ],
+      'events' => [
+        'name' => $this->t('Events'),
+        'data' => $this->getContent('events'),
+        'url' => '/search?keys=&f[0]=type%3Aevent',
+      ],
+    ];
+
+    return [
+      '#theme' => 'explore_block',
+      '#data' => $data,
+      '#cache' => [
+        'contexts' => $this->getCacheContexts(),
+        'tags' => $this->getCacheTags(),
+      ],
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheContexts() {
+    return Cache::mergeContexts(parent::getCacheContexts(), ['route']);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTags() {
+    $tags = [
+      'node:news',
+      'node:events',
+      'rdf_entity:solution',
+      'rdf_entity:collection',
+    ];
+
+    return Cache::mergeTags(parent::getCacheTags(), $tags);
+  }
+
+  /**
+   * Get entity data.
+   *
+   * @param string $type
+   *   The type of content.
+   *
+   * @return array
+   *   List of content with view mode "explore_item".
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  private function getContent($type): array {
+    $content = [];
+    $entity = $this->entityManager->getStorage('node');
+    $view_builder = $this->entityManager->getViewBuilder('node');
+    $query = $entity->getQuery();
+
+    $ids = $query->condition('status', 1)
+      ->condition('type', $type)
+      ->sort('created')
+      ->pager(12)
+      ->execute();
+
+    $entities = $entity->loadMultiple($ids);
+    foreach ($entities as $node) {
+      $content[] = $view_builder->view($node, $this->viewMode);
+    }
+
+    return $content;
+  }
+
+  /**
+   * Get rdf entity data.
+   *
+   * @param string $type
+   *   The type of content.
+   *
+   * @return array
+   *   List of rdf entity with view mode "explore_item".
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  private function getRdfEntity($type): array {
+    $rdf = [];
+    $entity = $this->entityManager->getStorage('rdf_entity');
+    $view_builder = $this->entityManager->getViewBuilder('rdf_entity');
+    $query = $entity->getQuery();
+
+    $ids = $query->condition('rid', $type)
+      ->sort('created')
+      ->pager(12)
```

Comment:
Here also, better to use `->range(0, 12)` because using a pager will needlessly keep track of the pager state in the `PagerManager`.